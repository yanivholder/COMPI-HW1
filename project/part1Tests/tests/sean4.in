void string "after \n\t comes an error \g yeah" not getting here