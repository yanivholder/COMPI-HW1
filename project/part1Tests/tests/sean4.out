1 VOID void
1 ID string
Error undefined escape sequence g
